### PR TITLE
Release/mui v6 to dev merge

### PR DESCRIPTION
**Pull request by shubham-eaton**
Fixes # .
https://github.com/etn-ccis/blui-react-workflows/issues/684

#### Changes proposed in this Pull Request:

-
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/package.json [hunk 1]
```diff
@@ -28,15 +28,15 @@
     "dependencies": {
         "@okta/okta-auth-js": "^7.7.0",
         "@okta/okta-react": "^6.9.0",
-        "dompurify": "^3.1.7",
+        "dompurify": "^2.2.9",
         "i18next-browser-languagedetector": "^8.0.0"
     },
     "peerDependencies": {
-        "@brightlayer-ui/react-components": "^6.1.2",
+        "@brightlayer-ui/react-components": "7.0.0-alpha.0",
```
Comment (JeffGreiner-eaton) [inline login-workflow/package.json:35]: use version 7.0.0

File: login-workflow/package.json [hunk 2]
```diff
@@ -75,18 +75,18 @@
         "@babel/preset-typescript": "^7.14.5",
         "@brightlayer-ui/eslint-config": "^3.0.0",
         "@brightlayer-ui/prettier-config": "^1.0.3",
-        "@brightlayer-ui/react-components": "^6.1.2-alpha.0",
-        "@brightlayer-ui/react-themes": "^7.1.0-alpha.0",
+        "@brightlayer-ui/react-components": "^7.0.0-alpha.0",
```
Comment (JeffGreiner-eaton) [inline login-workflow/package.json:78]: use version 7.0.0

File: login-workflow/package.json [hunk 3]
```diff
@@ -75,18 +75,18 @@
...
+        "@brightlayer-ui/react-themes": "8.0.0-alpha.0",
```
Comment (JeffGreiner-eaton) [inline login-workflow/package.json:79]: use version 8.0.0

File: login-workflow/example/package.json [hunk 4]
```diff
@@ -34,12 +34,12 @@
     "dependencies": {
         "@brightlayer-ui/colors": "^3.0.1",
         "@brightlayer-ui/icons-mui": "^3.6.0",
-        "@brightlayer-ui/react-components": "^6.1.2",
-        "@brightlayer-ui/react-themes": "^7.2.0",
+        "@brightlayer-ui/react-components": "7.0.0-alpha.0",
```
Comment (JeffGreiner-eaton) [inline login-workflow/example/package.json:37]: use version 7.0.0

File: login-workflow/example/package.json [hunk 5]
```diff
@@ -34,12 +34,12 @@
...
+        "@brightlayer-ui/react-themes": "^8.0.0-alpha.0",
```
Comment (JeffGreiner-eaton) [inline login-workflow/example/package.json:38]: use version 8.0.0

File: login-workflow/package.json (hunk 1 above)
Comment (shubham-eaton) [reply login-workflow/package.json]: updated to v7.0.0

File: login-workflow/package.json (hunk 2 above)
Comment (shubham-eaton) [reply login-workflow/package.json:78]: updated to v7.0.0

File: login-workflow/example/package.json (hunk 4 above)
Comment (shubham-eaton) [reply login-workflow/example/package.json]: updated to v7.0.0

File: login-workflow/package.json (hunk 3 above)
Comment (shubham-eaton) [reply login-workflow/package.json]: updated to v8.0.0

**Comment by JeffGreiner-eaton** [comment]
anyone else see this warning on change password submit? Login > change password

image
image

**Comment by shubham-eaton** [comment]
> anyone else see this warning on change password submit? Login > change password
> 
> image image

@JeffGreiner-eaton  we will take this up on next sprint 
https://github.com/etn-ccis/blui-react-workflows/issues/684

**Comment by joebochill** [comment]
Let's fix it today — we don't want console errors in the published package. It should only take a minute to fix the spread operator.

**Review by surajeaton — DISMISSED** [review]

**Review by JeffGreiner-eaton — APPROVED** [review]

**Pull request merged**